assets/airports/kjfk.json

```diff
@@ -7,227 +8,565 @@
+        "WITCH": ["N42d41.00m0", "W070d52.77m0"],
+        "YAHOO": ["N41d04.15m0", "W069d44.77m0"],
+        "YAZUU": ["N39d24.75m0", "W074d01.03m0"],
+        "YNKEE": ["N40.4835", "W73.849167"],
+        "ZACHS": ["N40.528802778", "W73.55386388888889"],
+        "ZALPO": ["N40.722030556", "W73.694752778"],
+        "ZETAL": ["N40.4899444", "W73.875925"],
+        "ZIZZI": ["N38d56.44m0", "W074d31.74m0"],
+        "ZULAB": ["N40.596663889", "W73.655736111"]
     },
+    "restricted": [
+    ],
     "runways":[
         {
-            "name":        ["13R", "31L"],
-            "name_offset": [[0, 0], [0.5, -0.5]],
-            "end":    [["N40.648333", "W73.816667"], ["N40.628", "W73.771833"]],
-            "ils":         [false, true]
+            "name": ["4L", "22R"],
+            "end": [
+                [40.622031, -73.785589, "11.9ft"],
+                [40.650518, -73.763326, "12.7ft"]
+            ],
+            "ils": [true, true]
         },
         {
-            "name":        ["4R", "22L"],
-            "name_offset": [[0, 0], [0, 0]],
-            "end":    [["N40.62542555", "W73.7703472167"], ["N40.645235833", "W73.7548636167"]],
-            "ils":         [true, true]
+            "name": ["4R", "22L"],
+            "end": [
+                [40.625461, -73.770333, "11.8ft"],
+                [40.645220, -73.754885, "11.8ft"]
+            ],
+            "ils": [true, true]
         },
         {
-            "name":        ["13L", "31R"],
-            "name_offset": [[0, 0], [0.5, -0.5]],
-            "end":    [["N40.6577648", "W73.790239533"], ["N40.6437261167", "W73.75927695"]],
-            "ils":         [true, true]
+            "name": ["13L", "31R"],
+            "end": [
+                [40.657779, -73.790243, "13.0ft"],
+                [40.643733, -73.759278, "12.7ft"]
+            ],
+            "ils": [true, true]
         },
         {
-            "name":        ["4L", "22R"],
-            "name_offset": [[0, 0], [0, 0]],
-            "end":    [["N40.622", "W73.785667"], ["N40.6505", "W73.763333"]],
-            "ils":         [true, true]
+            "name": ["13R", "31L"],
+            "end": [
+                [40.648370, -73.816718, "12.5ft"],
+                [40.628003, -73.771789, "12.5ft"]
+            ],
+            "ils": [false, true]
         }
     ],
+    "airways": {
+        "J174": ["HTO", "RIFLE", "GEDIC", "EMJAY", "YAZUU", "ZIZZI", "WARNN", "SWL"],
+        "J230": ["RBV", "COPES", "BYRDD", "SAAME", "LARRI", "BEETS", "VINSE", "LEJOY", "AIR"],
+        "J60": ["SAX", "CANDR", "NEWEL", "GYNTS", "DANNR", "FORTT", "DOOTH", "PSB"],
+        "J75": ["BOS", "SWALO", "NELIE", "GREKI", "CMK", "DUEYS", "JERSY", "SBJ", "BIGGY", "COPES", "MXE", "STOEN", "SACRI", "MURPH", "BINKS", "GVE"],
+        "Q818": ["GAYEL", "MSLIN", "STOMP", "BUFFY", "CFB"]
+    },
     "sids": {
-        "JFK3": {
-            "icao": "JFK3",
-            "name": "KENNEDY THREE",
+        "JFK5": {
```

Comment:
Kennedy 5 departure should be defined below the HAPIE6 departure

Reply:
done